Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/CustomFilePickerFragment.java

```diff
@@ -21,7 +21,7 @@
 public class CustomFilePickerFragment extends FilePickerFragment
 {
   private static final Set<String> extensions = new HashSet<>(Arrays.asList(
-          "gcm", "tgc", "iso", "ciso", "gcz", "wbfs", "wad", "dol", "elf", "dff"));
+          "gcm", "tgc", "iso", "ciso", "gcz", "wbfs", "wad", "dol", "elf", "dff", "raw"));
```

Comment:
I'm assuming you did this because it's needed for the SD card path, but this set of extensions is also used for selecting a game to play. If we're going to filter by extension, which extensions are allowed should be dependent on what you're using the file picker for.